week 2/script.js

```diff
@@ -0,0 +1,232 @@
+let currencyRates = {
+    timestamp: 1519296206,
+    base: "EUR",
+    date: "2021-03-17",
+    rates: {
+        USD: 1.23396
+    }
+};
+
+class CurrencyRate {
+    constructor(code, rate, date) {
+        this.code = code;
+        this.rate = rate;
+        this.date = date;
+    }
+}
+
+document.addEventListener('DOMContentLoaded', function() {
+    var coll = document.querySelectorAll(".collapsible");
```

Comment:
Prefer `let` or `const` instead of `var` so you can have more control over the scope of your variables.